package.json

```diff
@@ -7,19 +7,21 @@
 		"build": "next build",
 		"start": "next start",
 		"lint": "next lint",
-		"format": "prettier . --write"
+		"format": "prettier . --write",
+		"cy:open": "cypress open"
 	},
 	"dependencies": {
 		"@emotion/styled": "^11.14.0",
 		"@mui/material": "^6.4.4",
+		"@react-icons/all-files": "https://github.com/react-icons/react-icons/releases/download/v5.5.0/react-icons-all-files-5.5.0.tgz",
```

Comment:
Could you provide me some notes about the switch to `@react-icons/all-files"? I imagine it's the cypress conflict you reference in your commit, just curious as to what the compatibility error may be. 
Thanks!

Reply:
Absolutely. Sorry for not including it in the original description. [Here's a link](https://github.com/react-icons/react-icons/issues/934)